Makefile

```diff
@@ -99,125 +99,135 @@ pkgmap.peer           := $(PKGNAME)/cmd/peer
 include docker-env.mk
 include gotools.mk
 
+.PHONY: help
+# List all commands with documentation
+help: ## List all commands with documentation
+	@echo "Available commands:"
+	@awk 'BEGIN {FS = ":.*?## "}; /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
oh i didn't know you can specify more than 1 character as the field separator, nice.